bin/tessel-2.js

```diff
@@ -172,13 +210,21 @@ parser.command('key')
     choices: ['generate'],
   })
   .callback(function(opts) {
+    logs.verbose = opts.verbose;
     key(opts)
       .then(function() {
         logs.info('Key successfully generated.');
       })
       .then(closeSuccessfulCommand, closeFailedCommand);
+  })
+  .option('verbose', {
```

Comment:
It seems like many commands use these same settings. Could you refactor the 
`{
   default: false,
   flag: true,
   abbr: 'v',
    help: verbose
}`
object into it's own variable and assign it to each `'verbose'` option?